### ISSUE TITLE
Make fails with multiple "not declared" errors

### ISSUE THREAD

**Issue by Ma94xx** [libxls/libxls#127]
I've got the same issue as described in: https://github.com/libxls/libxls/issues/125.
Sadly there was no solution presented, since it was closed with "no longer interested".

**Comment by gleb-kun** [comment]
Hello.  I have a similar problem when using the stable version 1.6.2.  I'll try to figure it out soon and report my results here.

**Comment by gleb-kun** [comment]
Ok, I finally got around to this issue.

In file `xlstypes.h`:
```
#ifdef __cplusplus
#include <cstdint>
#else
#include <stdint.h>
#endif
```

You need to replace it with this:
```
#include <stdint.h>
```

Then you need to rebuild the configuration and build the project:
```
./configure
make
make install
```


In general, this problem is very easy to Google:
https://stackoverflow.com/questions/60116761/cstdint-string-cc11-int-usr-include-c-8-cstdint5311-error-int-fast8

**Comment by evanmiller** [comment]
PR welcome!

**Comment by QuLogic** [comment]
Should be fixed by #118.
